app/views/groups/_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<% if @group.errors.any? %>
+  <ul>
+    <% @group.errors.full_messages.each do |msg| %>
+       <li class="error"><%= msg %></li>
+    <% end %>
+  </ul>
+<% end %>
+
+<h3>Make a New Group</h3>
+<!-- Steven, Since I use this same form for the edit and new route, is there a good way i can display 'make a new group' if arriving from the 'new' route and 'Edit your Group' if arriving here from the 'edit' route? -->
+<%= form_for @group do |f| %>
```

Comment:
Ideally you might pull it out into a partial and have the label outside the partial. Alternatively, do 
<% if @group.new_record? %>
Make a new group
<% else %>
Edit your group
<% end %>

new_record? is something ActiveRecord::Base supplies